avm/utilities/tools/helper/Set-ModuleFileAndFolderSetup.ps1

```diff
@@ -46,11 +46,17 @@ function Set-ModuleFileAndFolderSetup {
     )
 
     if ([String]::IsNullOrEmpty($CurrentLevelFolderPath)) {
-        # First invocation. Handling provider namespace
-        $resourceTypeIdentifier = ($FullModuleFolderPath -split '[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}')[2] # avm/res/<provider>/<resourceType>
+        # Extract resource type identifier
+        $resourceTypeIdentifier = ($FullModuleFolderPath -split '[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}')[2] # <provider>\<resourceType>
```

Comment:
HEy @ChrisSidebotham, want to consider utls here too? Though to be fair I'd be curious to see if we'd have e.g. a `waf-aligned` test 😄

Reply:
To be fair, we have many places where this is the case and utl are not introduced yet. I'd suggest to not mix that up and take it later for the whole repo as part of the PR adding their support

Reply:
@AlexanderSehr - Lets leave this to a wider set-avmmodule utl allignment PR?

Reply:
I don't particularly care. Figured it would be good to immediately do it right here instead of knowingly adding outdated code, but not going to start a discussion over it if there are contrary opinons. Will resolve